tools/cloud_functions/bq_table_snapshots/bq_backup_create_snapshots.py

```diff
@@ -14,25 +15,36 @@
 BQ_JOBS_PROJECT_ID = ""
 
 
+def get_snapshot_timestamp(message):
+    cron_format = message['crontab_format']
+    it = CronSim(cron_format, datetime.now())
+    next_interval = next(it)
+    next_next_interval = next(it)
+    delta = (next_next_interval - next_interval).total_seconds()
+    prev_interval = next_interval - relativedelta(seconds=delta)
+    prev_cron_interval_timestamp = int(prev_interval.timestamp() * 1000)
+    return prev_cron_interval_timestamp 
+
+
 def get_bq_client():
```

Comment:
Make the client a global variable to allow reuse across multiple invocations https://cloud.google.com/functions/docs/bestpractices/tips#use_global_variables_to_reuse_objects_in_future_invocations

Reply:
done